3MI0800331/Task2.cpp

```diff
@@ -0,0 +1,76 @@
+#include <iostream>
+using namespace std;
+
+
+struct orderedPair {
+	int first;
+	int second;
+};
+
+void print(const orderedPair& arg) {
+		cout << arg.first << " " << arg.second;
+};
+
+orderedPair compare(const orderedPair& arg1, const orderedPair& arg2) {
+	orderedPair greatest;
+	if (arg1.first > arg2.first) {
```

Comment:
Why not simply return arg1 or arg2 :thinking: Anyways when writing a function named compare there's a rule of thumb that one can see pretty much everywhere and it says that a `compare` function takes two arguments and returns:
- `negative integer ` if `arg1 < arg2`
- `zero` if `arg1 == arg2`
- `positive integer` if `arg1 > arg2`